.github/workflows/verify-version.yml

```diff
@@ -0,0 +1,35 @@
+name: Verify plugin version
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  verify-version:
+    name: Verify plugin version
```

Comment:
What's that workflow? Let's document its purpose inline or give it a name that explains the intent